db/seeds.rb

```diff
@@ -175,6 +175,13 @@ def setup_application_instances(application, application_instances)
   end
 end
 
+BundleInstance.find_each do |bundle_instance|
+  bundle_instance.applications.each do |app|
+    instance = ApplicationInstance.find_by(site: bundle_instance.site, application: app)
```

Comment:
consider storing the site to a variable outside this `each`
also could write this as
```ruby
instance = app.application_instances.find_by(site: site)
```